### PR TITLE
Readme for building and running Testbench

### PR DESCRIPTION
**Pull request by pdavis68**
Readme with details for building and running Testbench

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,119 @@
...
+pip install -r requirements.txt
+cd app
+npm install
+```
+
+Create a .env file in the langchain-testbench folder with your api keys and root pw.
+
+```.env
+MONGO_ROOT_PASSWORD=[mongo root pw]
+OPENAI_API_KEY=[your openai API key]
```
[the remark this answers is not in the document]
Comment (pdavis68) [reply README.md:89]: The .env files are in the .gitignore file.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,119 @@
-# testbench
-A system for designing and testing chains
+
+## Pre-requisites
+
+- OpenAI API key
+- Hugging Face API Key
```
[the remark this answers is not in the document]
Comment (pdavis68) [reply README.md:5]: I'm confused. You don't need either? Or do you need at least one of them?

[the remark this answers is not in the document]
Comment (pdavis68) [reply README.md]: Modified the documentation around this.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,119 @@
...
+
+- Create a hugging face account (https://huggingface.co)
+- Generate an API key  (https://huggingface.co/settings/tokens)
+- Copy the key somewhere safe
+
+It's good to create separate API keys for separate projects/tools. That way if you need to disable a key for a project, you don't have to generate new keys for all your other projects.
+
+## Gitting the code
+
+https://github.com/wm-pxel/langchain-testbench
```
[the remark this answers is not in the document]
Comment (pdavis68) [reply README.md]: Removed it. Originally wrote the documentation in Obsidian and just copied and pasted it. The URL was for my reference at the time.

**Review by acruikshank-wm — APPROVED** [review]
Looks great.

**Pull request merged**